presto-resource-group-managers/src/test/java/com/facebook/presto/resourceGroups/reloading/TestReloadingResourceGroupConfigurationManager.java

```diff
@@ -230,7 +239,8 @@ private static void assertEqualsResourceGroup(
             int schedulingWeight,
             boolean jmxExport,
             Duration softCpuLimit,
-            Duration hardCpuLimit)
+            Duration hardCpuLimit,
+            ResourceGroupQueryLimits resourceGroupQueryLimits)
```

Comment:
add at the end of this function `assertEquals(group.getPerQueryLimits(), resourceGroupQueryLimits);`